ldap3/extend/standard/readControl.py

```diff
@@ -0,0 +1,57 @@
+"""
+"""
+
+# Created on 2016.12.19
+#
+# Author: Giovanni Cannata
```

Comment:
I thought of that; there's no other author statements but his name, I presume because he is the primary author and maintainer of ldap3.